utility/service/service.go

```diff
@@ -350,3 +548,44 @@ func (r *relay) GetRelayChain() RelayChain              { return nil }
 func (r *relay) GetGeoZone() GeoZone                    { return nil }
 func (r *relay) GetToken() AAT                          { return nil }
 func (r *relay) GetSignature() string                   { return "" }
+
+// executeHTTPRequest performs the HTTP request that sends the relay to the chain's URL.
+func executeHTTPRequest(cfg *configs.ChainConfig, relay *coreTypes.RelayPayload) (*coreTypes.RelayResponse, error) {
```

Comment:
Not much to add here. If we point to the Pocket V1 blochcain & localhost instead of ETH (i.e. 0021) with a fake URL, it might be an even easier way to test things E2E.

Reply:
Thank you, fixed.